lib/Elastica/Query/Filtered.php

```diff
@@ -21,6 +23,8 @@ class Filtered extends AbstractQuery
      */
     public function __construct(AbstractQuery $query = null, AbstractFilter $filter = null)
     {
+        trigger_error('Use BoolQuery instead. Filtered query is deprecated since ES 2.0.0-beta1 and this class will be removed in further Elastica releases.', E_USER_DEPRECATED);
```

Comment:
I have tried it but it's not possible to make `FilteredTest` to work then. This is the only way how I was able to make tests work.